stories/shared/EvaluationForm/index.jsx

```diff
@@ -0,0 +1,36 @@
+import { Unstyled } from '@storybook/addon-docs';
+import Accordion from '../../../components/Accordion';
+
+const urlGenerator = (componentName) =>
+  `https://docs.google.com/forms/d/e/1FAIpQLSewdS_VhSt_JSFOxiuiIc-auEj0MQwn_P_WiXYhAEAE1DqJhg/viewform?entry.1112120969=${componentName}`;
+
+const EvaluationForm = ({ componentName }) => (
+  <Unstyled>
+    <Accordion
+      items={[
+        {
+          content: (
+            <ul>
+              <li>
+                <iframe
+                  title="Rate our Documentation"
+                  src={`${urlGenerator(componentName)}`}
+                  width="992"
+                  height="600"
+                  frameBorder="0"
+                  marginHeight="0"
+                  marginWidth="0"
+                >
+                  Carregando…
+                </iframe>
+              </li>
+            </ul>
+          ),
+          title: '⭐⭐⭐⭐⭐ Rate our Documentation',
```

Comment:
What do you think about changing the text and leaving it like the image below? It seems to me that the text before the icons provides a more consistent visual structure

![image](https://github.com/catho/quantum/assets/105747728/7859db39-9641-476f-b9d4-7bc8ffc7dc93)
![image](https://github.com/catho/quantum/assets/105747728/78da2bef-5eb3-4a0f-b25e-0cc318d866fc)